flask/helpers.py

```diff
@@ -580,9 +581,13 @@ def send_file(filename_or_fp, mimetype=None, as_attachment=False,
     rv.cache_control.public = True
     if cache_timeout is None:
         cache_timeout = current_app.get_send_file_max_age(filename)
-    if cache_timeout is not None:
+    if cache_timeout is not None and cache_timeout > 0:
         rv.cache_control.max_age = cache_timeout
         rv.expires = int(time() + cache_timeout)
+    else:
```

Comment:
I probably agree, but am unsure what the other parameters would actually change if the timeout is already `0`.